docker/batch-single/README.md

```diff
@@ -65,12 +65,18 @@ To run the Batch Single Docker container, copy the `run.sh.example` script to a
 | `S3_SUBMISSION_PREFIX`     	 | The prefix (directory) where the extracted submission ttl files are located | 
 | `S3_SUBMISSION_VALIDATION_DESCR` | The description of the validation type that will be performed on the submission |       
 | `AWS_BATCH_JOB_ID`       | The job ID assigned for this validation from AWS Batch | 
-| `AWS_BATCH_JOB_NODE_INDEX`       | The node ID assigned to current node from AWS Batch |    
-| `MAIN_SLEEP_INTERVAL`        	 | Interval for the main to monitor the SQS queue for depletion |  
+| `AWS_BATCH_JOB_NODE_INDEX`       | The node ID assigned to current node from AWS Batch |  
```

Comment:
Another minor thing-- could you fix the `The The` and the `Validatior` typos in the `Batch Single` section above?

Reply:
Updated.